packages/app-extensions/src/form/syncValidation.js

```diff
@@ -11,7 +11,7 @@ export default entityModel =>
     )
   )
 
-const addErrors = (errors, field, fieldErrors) => (
+export const addErrors = (errors, field, fieldErrors) => (
```

Comment:
since this method is used in async validation i would move it out of this file

Reply:
It is used in both files: 
syncValidation.js Line 44, 72
asyncValidation.js Line 39
Should I move it to a third location?

Reply:
that what i meant , yes